apps/store/themes/store/css/theme-categorization.min.css

```diff
@@ -0,0 +1,136 @@
+.radio .helper:before,.radio>input[type=radio]:checked+.helper:after,.switch .helper:before,.switch>input[type=checkbox]+.helper:after,.tree-view.tree-view-lines ul li:before,.tree-view.tree-view-lines ul:before {
```

Comment:
Thanks for pointing it out. Fixed in https://github.com/wso2/carbon-store/commit/0424453c4bc5e31408bc123636add41c604a4ef1